proposals/Exceptions.md

```diff
@@ -55,9 +43,9 @@ by a new _exception_ section of a WebAssembly module. The exception section is a
 list of exceptions. Each exception has a _type signature_. The type signature
 defines the list of values associated with an exception.
 
-Within the module, exceptions are identified by an index into
-the [exception index space](#exception-index-space).  This index is referred to
-as the _exception tag_.
+Within the module, exceptions are identified by an index into the [exception
+index space](#exception-index-space). This index is referred to as the
```

Comment:
Old text, but worth fixing while you're here: the index is not the tag, the index references the tag.